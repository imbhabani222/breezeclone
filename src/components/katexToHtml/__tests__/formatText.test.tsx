import React from "react";
import { cleanup, render, screen } from "@testing-library/react";
import FormatText from "../formatText";

describe("formatText Testing", () => {
  function renderFormatText() {
    return render(<FormatText />);
  }

  afterEach(cleanup);
  test("Render without Crashing", () => {
    renderFormatText();
  });
  test("Match snapshot footer component", () => {
    const { asFragment } = renderFormatText();
    expect(asFragment()).toMatchSnapshot();
  });
});
