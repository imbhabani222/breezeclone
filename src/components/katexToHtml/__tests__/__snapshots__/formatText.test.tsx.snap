// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`formatText Testing Match snapshot footer component 1`] = `
<DocumentFragment>
  <p
    class="optionClass"
    data-testid="testid"
  />
</DocumentFragment>
`;
